modules/core/arrow-extras-data/src/main/kotlin/arrow/data/SortedMapK.kt

```diff
@@ -49,7 +50,7 @@ data class SortedMapK<A : Comparable<A>, B>(private val map: SortedMap<A, B>) :
 
   fun <G, C> traverse(GA: Applicative<G>, f: (B) -> Kind<G, C>): Kind<G, SortedMapK<A, C>> = GA.run {
     map.iterator().iterateRight(Eval.always { just(sortedMapOf<A, C>().k()) }) { kv, lbuf ->
-      f(kv.value).map2Eval(lbuf) { (mapOf(kv.key to it.a).k() + it.b).toSortedMap().k() }
+      f(kv.value).map2Eval(lbuf) { (mapOf(kv.key to it.a).mk() + it.b).toSortedMap().k() }
```

Comment:
mk?

Reply:
it was necessary the file has multiple k() function from different imports